### ISSUE TITLE
Run selector mismanages state of closed panes

### ISSUE THREAD

**Issue by marksandler2** [tensorflow/tensorboard#884]
0. Open tensorboard containing multiple runs/panes
1. Go to image tab
2. Click on a run's radiobox
3. Toggle runs checkbox a few time
4. Try to expand any of the categories, will see multiple duplicates of the same run.

**Comment by nfelt** [comment]
FWIW this is also reproducible with the histograms dashboard.  The "detached" histograms are there after the run is deselected the first time, and then they become duplicates after the run is re-selected.  The "detached" charts also don't respond to the other sidebar controls ("Histogram mode" and "Offset time axis").

**Comment by nfelt** [comment]
I'm seeing a probably related issue with the scalar and PR curve dashboards, but instead where the run selector mismanages *open* panes.  Any open panes get "shrunk" down to an unreadable size after you navigate away from that dashboard, click any run in the run selector, and then navigate back.

Exact steps to reproduce:
1. visit the relevant dashboard (scalars or PR curves) with data displayed
2. navigate to some other dashboard
3. click on any run in the run selector
4. navigate back to original dashboard

The charts should be unchanged on the original dashboard, but instead they appear shrunk as shown below (for the first two categories; the temperature category was not originally open, and hence was not shrunk, when I opened it after step 4 above, for comparison purposes).  Clicking any run in the run selector will restore them to their original size.

![image](https://user-images.githubusercontent.com/710113/35251868-3ce2ffb0-ff92-11e7-8367-fa740122d182.png)

**Comment by chihuahua** [comment]
Ah, indeed, we should fix that. A while back, we thought about fixing this on the Plottable side, but that thread unfortunately went nowhere: 
https://github.com/tensorflow/tensorboard/issues/554

Several brain residents have noticed that issue.

**Comment by nfelt** [comment]
Oh okay, so that is a separate issue.  Sorry for muddying up this issue thread!

**Comment by chihuahua** [comment]
It's a good thing you resurfaced it! We really ought to fix it.

**Comment by nfelt** [comment]
It turns out this adds duplicate cards to all pages, not just the first one.  In particular I see the following behavior when using the images_demo data that has 4 samples of the box_to_gaussian tag:

1. Open images dashboard
2. Change pagination limit to 2 (forcing samples 1 and 2 onto first page, 3 and 4 onto second page)
3. Close 'box_to_gaussian' category pane
4. Uncheck, then re-check the 'box_to_gaussian' run in the runs selector
5. Expand 'box_to_gaussian' category pane

The expected behavior is that it looks the same as it did after step 2, i.e. it looks the same.  However, what I see instead is that now each page has 4 cards, where the first two cards on each page are duplicate copies of the sample 1 and sample 2 cards (corresponding to the cards that were on the first page) and the second two cards are the expected ones.

Notably, the first two cards that were added appear to be "detached" in that they're unresponsive the brightness and contrast sliders.  The second two cards still respond to the sliders as expected.

If steps 3-5 are repeated, another set of duplicate cards will be added in just above the last set (which remains the "live" set).  It's possible to detect that they are added last rather than first because the new duplicates that appear seem to always be tied to whatever page was visible at the time that the category pane was closed.  E.g. if I navigate to page 2 as a step 2.5 above, and then do steps 3-5, the duplicate "dead" cards that appear are for samples 3 and 4 on both pages, rather than samples 1 and 2.

Here's an illustration of a typical result:

```
page 1
1 of 4 | 2 of 4 (1st copy)
1 of 4 | 2 of 4 (2nd copy)
3 of 4 | 4 of 4 (3rd copy after moving to page 2 before closing pane)
1 of 4 | 2 of 4 (live)

page 2
1 of 4 | 2 of 4 (1st copy)
1 of 4 | 2 of 4 (2nd copy)
3 of 4 | 4 of 4 (3rd copy after moving to page 2 before closing pane)
3 of 4 | 4 of 4 (live)
```

**Comment by nfelt** [comment]
I followed up on this internally (see g/polymerdiscuss/KxlI4B8FXUE) and it appears to be due to this issue in Polymer: https://github.com/Polymer/polymer/issues/5168

There's a workaround that involves changing the category pane to remove `restamp` from its dom-if and wrapping its `<slot>` in a `<span>` element.  However, removing `restamp` causes chart-shrinkage issues, so those would need to be solved to avoid trading one problem for another.

cc @stephanwlee FYI

**Comment by bileschi** [comment]
This appears to be resolved, (at least following Nick's instructions in #issuecomment-372480462 )
